testdata/go/v3/memcached-operator/monitoring/metrics.go

```diff
@@ -1,23 +0,0 @@
-package monitoring
```

Comment:
@NikhilSharmaWe this one should not be removed.
It was introduced in the PR: https://github.com/operator-framework/operator-sdk/pull/5972

You probably remove its implementation in the rebase with the master.

Reply:
/hold